core-bundle/contao/themes/flexible/basic.css

```diff
@@ -813,12 +813,16 @@ a.tl_submit {
 	display: table;
 }
 
-.w50 {
+.w25, .w33, .w50, .w66, .w75 {
```

Comment:
I think `.w66` and `.w75` should fall back to 100% with in non-large screens.

```suggestion
.w25, .w33, .w50 {
```

Reply:
No, that will not work (see the first form example). The fields have to behave backwards compatible.

Reply:
<img width="1058" alt="" src="https://user-images.githubusercontent.com/1192057/212125345-f90e067e-60d6-4f13-9084-3f5ab7eece38.png">

With 100% width, it would fall-back to this:

<img width="1013" alt="" src="https://user-images.githubusercontent.com/1192057/212125434-eacd37d3-97fc-4775-8b76-142cdcb4b625.png">

However, this is how it looks in Contao < 5.1:

<img width="1019" alt="" src="https://user-images.githubusercontent.com/1192057/212125531-d7fc6278-a4b5-4717-9bea-da8fe4f7c2b0.png">